specification/cognitiveservices/data-plane/LUIS/Authoring/preview/v3.0/LUIS-Authoring.json

```diff
@@ -7759,6 +7780,13 @@
         "parameters": [
           {
             "$ref": "#/parameters/BearerAuthPath"
+          },
+          {
+            "in": "header",
```

Comment:
I would suggest to make it a parameter in the `parameters` section and ref it here like `"$ref": "#/parameters/ArmTokenParamter"` so you can re-use it consistently everywhere needed